tests/Integration/Validation/ValidationTest.php

```diff
@@ -90,6 +92,47 @@ public function testFullValidationError(): void
             ]);
     }
 
+    public function testFullValidationErrorWithoutLocationParse(): void
+    {
+        /** @var ConfigRepository $config */
```

Comment:
Changed using the `$app` so that will be automatically hinted.
Changes here https://github.com/nuwave/lighthouse/pull/2499/commits/3b009b9c608445ed12f8284c31ca77155f970536